src/pkg/packager/create.go

```diff
@@ -41,8 +41,10 @@ func (p *Packager) Create(baseDir string) error {
 	}
 
 	// Load the images and repos from the 'reference' package
-	if err := p.loadDifferentialData(); err != nil {
-		return err
+	if p.cfg.CreateOpts.DifferentialData.DifferentialPackagePath != "" {
+		if err := p.loadDifferentialData(); err != nil {
+			return err
+		}
```

Comment:
I prefer to have one `nil` check (which one already exists in `loadDifferentialData`) since over time it can be confusing as to which function is responsible for that check - if this were used in multiple places and someone did not check all of them but decided to remove the check from the sub function then that could break things if they weren't careful.